test/unit/components/message.js

```diff
@@ -35,6 +35,10 @@ describe('<FormattedMessage>', () => {
         );
     });
 
+    it('should not throw when <IntlProvider> is missing from ancestry but has defaultMessage', () => {
+        expect(() => renderer.render(<FormattedMessage id="hello" defaultMessage="Hello" />)).toNotThrow();
```

Comment:
It would safer to check that "Hello" is properly rendered.

Reply:
Addressed :)